lib/containers/container_images.pm

```diff
@@ -57,6 +57,11 @@ sub build_and_run_image {
     record_info('Downloading', "Dockerfile: containers/$dockerfile\nHTML: containers/index.html");
     assert_script_run "mkdir -p $dir/BuildTest";
     assert_script_run "curl -f -v " . data_url("containers/$dockerfile") . " > $dir/BuildTest/Dockerfile";
+    if ($dockerfile eq 'Dockerfile.python3') {
```

Comment:
For now ok, but in the future we should move away from hardcoded filenames and bundle the test containers in their own directory or tarball and set the Dockerfile as variable.

This is beyond the scope of this PR though.

Reply:
> For now ok, but in the future we should move away from hardcoded filenames and bundle the test containers in their own directory or tarball and set the Dockerfile as variable.

i agree. hardcoded strings do not look good. but it was straight forward in this case. it is interesting to me how this can improved. 

> 
> This is beyond the scope of this PR though.

thank you